### PR TITLE
Pundit policies - Coffee controller

### PR DESCRIPTION
**Pull request by ttomz12**
Done for the coffee controller but we can add it to other features if we think it's necessary

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,6 +1,16 @@
 class ApplicationController < ActionController::Base
-  # strong params fo rdevise is on application controller 
-  before_action :configure_permitted_parameters, if: :devise_controller?
```
Comment (kwall925) [inline app/controllers/application_controller.rb:3]: this same code is also on line 3

File: app/controllers/coffees_controller.rb [hunk 2]
```diff
@@ -1,7 +1,9 @@
 class CoffeesController < ApplicationController
+  before_action :set_coffee, only: [:show, :edit, :update, :destroy]
 
   def index
-    @coffees = Coffee.all
```
Comment (kwall925) [inline app/controllers/coffees_controller.rb:4]: duplicate on the next line

File: app/controllers/coffees_controller.rb [hunk 3]
```diff
@@ -27,7 +31,8 @@ def edit
   end
 
   def update
-    @coffee = Coffee.find(params[:id])
```
Comment (kwall925) [inline app/controllers/coffees_controller.rb:31]: duplicate on the next line

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (ttomz12) [reply app/controllers/application_controller.rb:3]: weird maybe i didnt save it properly

Comment (ttomz12) [reply app/controllers/application_controller.rb:3]: wait nevermind it should be fine, not sure what you're referring to

File: app/controllers/coffees_controller.rb (hunk 2 above)
Comment (ttomz12) [reply app/controllers/coffees_controller.rb:4]: done

File: app/controllers/coffees_controller.rb (hunk 3 above)
Comment (ttomz12) [reply app/controllers/coffees_controller.rb:31]: done

**Review by ttomz12 — COMMENTED** [review]
think i finished everything

**Review by kwall925 — APPROVED** [review]

**Pull request merged**